bugsnag/asgi.py

```diff
@@ -100,6 +100,8 @@ def add_request_info(event):
             request['url'] = parse_url(request, server)
 
             event.add_tab("request", request)
+            if bugsnag.configure().send_environment:
+                event.add_tab("environment", scope)
```

Comment:
Should we add that we now collect the environment in ASGI apps to the changelog too?

Reply:
oh yeah, good call!